external/wakaama/examples/client/lwm2mclient.c

```diff
@@ -1209,48 +1210,55 @@ int lwm2m_client_main(int argc, char *argv[])
     if (NULL == objArray[1])
     {
         fprintf(stderr, "Failed to create server object\r\n");
+		close(data.sock);
         return -1;
```

Comment:
Please, check final source codes for external/wakaama/examples/client/lwm2mclient.c. Do not have these codes. We have changed back wakama to it's original state. I mean original opensource.